.changeset/grumpy-dryers-feel.md

```
@@ -0,0 +1,5 @@
+---
+"eridu-tech": patch
+---
+
+Updated dependency version
```

markdownlint-cli2 (0.23.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)